crates/mako/src/generate/runtime.rs

```diff
@@ -12,4 +12,5 @@ pub struct AppRuntimeTemplate {
     pub is_browser: bool,
     pub concatenate_enabled: bool,
     pub cross_origin_loading: Option<String>,
+    pub global_module_registry: bool,
```

Comment:
guess: how many fields will be hold in this struct ?

Reply:
I don't know ...